src/entry_names_gbdk.h

```diff
@@ -14,7 +14,12 @@
     // Tool
     DEF_NAME_STR(STR_GBDK, "GBDK");
     // Versions
-    DEF_NAME_STR(STR_GBDK_2_x_to_2020_3_2_0,       "2.x - 2020.3.2.0");
+    DEF_NAME_STR(STR_GBDK_1_x,                     "1.x");
+    DEF_NAME_STR(STR_GBDK_2_0_x_to_2_0_17,         "2.0.x - 2.0.17");
```

Comment:
I can't check what crt0 the pre-2.0.17 versions emitted due to missing builds, which is why this is used as a fallback.

Reply:
Looks good, that's still much improved coverage. :+1: